zig.mod

```diff
@@ -0,0 +1,4 @@
+id: xzxo5rnug8wjvplr5pxamvgegg989aqmli57pi8ah7p93kx8
+name: zig-args
```

Comment:
Would it be okay to remove the zig- prefix and only let "args" as a name ?

Reply:
I usually just call it `args`, the `zig-` prefix is just there to find it better